locale/templates/settings.ftl

```diff
@@ -247,19 +250,24 @@ delete-account-password-input =
  .label = Enter password
 
 delete-account-cancel-button = Cancel
-delete-account-delete-button = Delete Account
+delete-account-delete-button = Delete
```

Comment:
@6a68 oops, I overlooked this one. This needs to have a new string ID. Can we fix it ASAP?